mypy_django_plugin/transformers/models.py

```diff
@@ -243,13 +243,11 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
             except helpers.IncompleteDefnException as exc:
                 # Check if manager is a generated (dynamic class) manager
                 base_manager_fullname = helpers.get_class_fullname(manager.__class__.__bases__[0])
-                manager_info = self.get_generated_manager_info(manager_fullname, base_manager_fullname)
-                if manager_info is None:
+                if manager_fullname not in self.get_generated_manager_mappings(base_manager_fullname):
                     # Manager doesn't appear to be generated. Track that we encountered an
                     # incomplete definition and skip
-                    encountered_incomplete_manager_def = True
-                    continue
-                _, manager_class_name = manager_info.fullname.rsplit(".", maxsplit=1)
+                    incomplete_manager_defs.add(manager_name)
+                continue
```

Comment:
This changeset triggered some presumably premature assumption that dynamically generated managers can be looked up and have a type/typeinfo during semantic analysis (where they are prepared). That shouldn't be possible until the type checker has run. Which, I think, only happens after semantic analysis.